broker-daemon/models/order.js

```diff
@@ -246,6 +246,16 @@ class Order {
     }
   }
 
+  /**
+   * @return {Boolean}
+   */
+  get active () {
```

Comment:
why define this in this PR if we aren't using it?

Reply:
Good catch. I forgot to remove these from the order/fill.